mxcube3/routes/qutils.py

```diff
@@ -537,11 +542,12 @@ def set_dc_params(model, entry, task_data):
     acq.path_template.base_prefix = params['prefix']
 
     full_path = os.path.join(mxcube.session.get_base_image_directory(),
-                             params.get('path', ''))
+                             params.get('subdir', ''))
```

Comment:
Just for me to understand, what is the difference between `params.get('path')` and `params.get('subdir')` ?
Could you please show me with an example ?
It seems to me `full_path` won't be full path anymore with this change but I am not sure ?